.env.example

```
@@ -1,5 +1,12 @@
+# Rackula Configuration
 # Copy this file to .env and customize as needed
 
+# === Docker Configuration ===
 # Port to expose Rackula on (default: 8080)
 RACKULA_PORT=8080
+
+# === Local Development ===
+# Umami analytics configuration (optional)
+VITE_UMAMI_ENABLED=false
+VITE_UMAMI_WEBSITE_ID=
+VITE_UMAMI_SCRIPT_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_UMAMI_SCRIPT_URL key should go before the VITE_UMAMI_WEBSITE_ID key

(UnorderedKey)